docs/installation/opni/slo.md

```diff
@@ -3,5 +3,219 @@ title: Enable SLOs
 slug: /installation/opni/slo
 ---
 
-TODO Alex
-Introduce SLOs, how to set them up, etc.
\ No newline at end of file
+## Introduction
+
+The term [SLO](#service-level-objective-slo), or Service Level Objective, is derived from the need in production applications for SLAs, or Service Level Agreements.
+
+SLAs specify a target for application performance, reliability or health, and SLOs are the tool used to **observe**
+said application's performance, reliability or health.
+
+SLOs can be broken down into two components :
+
+- [SLI](#service-level-indicator-sli) : the actual metrics that represent the application performance, reliability or health over some **period** of time
+- Target : an objective we want the metrics the SLI defines to abide by
+
+## Opni Prerequisites
```

Comment:
maybe we can move prerequisites to right before or at the beginning of  `getting started` ? Everything else looks pretty clear and nice.

Reply:
addressed this. moved to beginning of `getting started`